lib/active_merchant/billing/gateways/opp.rb

```diff
@@ -335,6 +335,13 @@ def commit(post, authorization, options)
         )
       end
 
+      def headers
+        { 
```

Comment:
Rubocop is telling me there's a trailing whitespace here.

Reply:
Updated 👍